students/patchcarrier/Lesson04/mailroom2.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env python3
+
+donors = {"Conrad Anker":[550, 1200, 0.02], 
+          "Tommy Caldwell":[600.50, 80], 
+          "Margo Hayes":[200, 550.50], 
+          "Alex Honnold":[0.01],
+          "Paige Claassen":[750, 800, 150.25]}
+
+
+def prompt_user(prompt, acceptable_vals):
+    """Prompt the user for an input until the input is an element of acceptable_vals.
+    
+    :param prompt:          Prompt to display to the user with acceptable commands
+    :param acceptable_vale: An iterable of acceptable characters
+    """
+    
+    user_input = input(prompt)
+    
```

Comment:
You can remove a lot of the empty space in this program as well - blank rows inside of these functions aren't needed.